parametermanager/api/ParameterManager.Samples/README.md

```diff
@@ -0,0 +1,32 @@
+# .NET Parameter Manager Samples
```

Comment:
Can you move this file one level up so we don't need one for global and one for regional samples? I think the current ones are identical. Are you expecting them to differ in upcoming PRs?